src/Microsoft.ML.Data/DataLoadSave/EstimatorChain.cs

```diff
@@ -57,7 +57,8 @@ public EstimatorChain()
         public TransformerChain<TLastTransformer> Fit(IDataView input)
         {
             // Before fitting, run schema propagation.
-            GetOutputSchema(SchemaShape.Create(input.Schema));
+            SchemaShape schemaShape = SchemaShape.Create(input.Schema);
+            GetOutputSchema(schemaShape);
```

Comment:
Why is this changed?

Reply:
I had changed this line to see the difference in the stored annotations between input.Schema and schemaShape, and experimented in ways to store the annotations at this step in the code.